code/Employee.php

```diff
@@ -89,10 +89,18 @@ class Employee extends DataObject
 
     private static $table_name = 'Employee';
 
+    /**
+     * Use basic scaffold settings (no tabs, etc)
+     */
+    private static array $scaffold_cms_fields_settings = [
+        'includeRelations' => false,
+        'tabbed' => false,
+        'ajaxSafe' => false,
+    ];
+
     public function getCMSFields()
     {
-        // Use basic scaffolder (no tabs)
-        $fields = $this->scaffoldFormFields();
+        $fields = parent::getCMSFields();
```

Comment:
Need to call parent::getCMSFields() so that the new extension hook works

Reply:
Could have also just called `$this->extend('the-hook-name')` but this way is probably better practice :+1: